docs-source/spring/content/development/cli/_index.md

```diff
@@ -22,43 +22,49 @@ The platform specific binary can be renamed to `oractl` for convenience.
+       connect: Connect to the OBaaS Spring Cloud admin console.
 
 Application/Namespace Commands
-- `create`: Create an application/namespace.
-- `delete`: Delete a service or entire application/namespace.
+       create: Create an application/namespace.
+       delete: Delete a service or entire application/namespace.
 
 Built-In Commands
-- `help`: Display help about available commands
-- `stacktrace`: Display the full stacktrace of the last error.
-- `clear`: Clear the shell screen.
-- `quit`, `exit`: Exit the shell.
-- `history`: Display or save the history of previously run commands
-- `version`: Show version info
-- `script`: Read and execute commands from a file.
+       help: Display help about available commands
+       stacktrace: Display the full stacktrace of the last error.
+       clear: Clear the shell screen.
+       quit, exit: Exit the shell.
+       history: Display or save the history of previously run commands
+       version: Show version info
+       script: Read and execute commands from a file.
 
 Informational Commands
-- `list`: list/show details of application services.
+       list: list/show details of application services.
 
 Service Commands
-- `bind`: Create a schema/user and bind it to service deployment.
-- `config: View and modify application configuration.
-- `deploy`: Deploy a service.
+       bind: Create a schema/user and bind it to service deployment.
+       config: View and modify application configuration.
+       deploy: Deploy a service.
 
-An application is a namespace encompassing related microservices. For example, a "cloudbank" application may have "banktransfer", "frauddetection", etc. microservices deployed within it.
+oractl:>
+```
+
+An application is a namespace encompassing related microservices. For example, a "cloudbank" application may have "banktransfer" "frauddetection", etc. microservices deployed within it.
 The `create` command results in the creation of an application namespace.
 
 The `bind` command results in the automatic creation of a database schema for a given service/user and binds the information for that schema/database in the environment of the microservice for it to use.  The option of the prefix for the environment properties bound is also given.  For example, most Spring microservices us "spring.datasource".
 
 The `deploy` command takes `serviceName`, `appName`, and `jarLocation` as it's main arguments (`imageVersion` and `javaVersion` options are also provided).
-When the deploy command is issued, the microservice jar file is uploaded to the backend, a Docker image is created for the jar/microservice, and various Kubernetes resources such as deployment, service, etc. are also created.  
+When the deploy command is issued, the microservice jar file is uploaded to the backend, a Docker image is created for the jar/microservice, and various Kubernetes resources such as deployment, service, etc. are also created.
```

Comment:
nitpick: JAR is capitals.
we prefer to say "a container image" not "a Docker image" since that is a more inclusive term

Reply:
Fixed